hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/routes-v2.tsx

```diff
@@ -53,5 +55,13 @@ export const routesV2 = [
   {
     path: '/Containers',
     component: Containers
+  },
```

Comment:
And also the View Insights page in the overview page does not route us to the Insights page.

<img width="1160" alt="image" src="https://github.com/user-attachments/assets/d18f17ea-7f1c-428f-b876-6dea3f69cdb9">

Reply:
Addressed in [8add6c2](https://github.com/apache/ozone/pull/7327/commits/8add6c2fccf9b0e999fac47ef0a65fa19e996b2c)